tests/acceptance/job/basic-layout-test.js

```diff
@@ -64,6 +64,60 @@ test('visiting job-view', function (assert) {
   });
 });
 
+test('visiting pull request job-view', function (assert) {
+  let repo = server.create('repository', { slug: 'travis-ci/travis-web' }),
+    branch = server.create('branch', { name: 'acceptance-tests' });
+
+  let  gitUser = server.create('git-user', { name: 'Mr T' });
+  let commit = server.create('commit', { author: gitUser, committer: gitUser, branch: 'acceptance-tests', message: 'This is a message', branch_is_default: true });
+
+  let request = server.create('request', { pull_request_mergeable: 'draft' });
+  server.create('message', {
+    request,
+    level: 'info',
+    key: 'group',
+    code: 'flagged',
+    args: {
+      given: 'group'
+    }
+  });
+
+  let user = server.create('user', {
+    name: 'Mr T',
+    avatar_url: '/images/favicon-gray.png'
+  });
+
+  let build = server.create('build', { repository: repo, state: 'passed', createdBy: user, commit, branch, event_type: 'pull_request', pull_request_number: 1, request });
+  let job = server.create('job', { number: '1234.1', repository: repo, state: 'passed', build, commit });
+  commit.job = job;
+
+  job.save();
+  commit.save();
+
+  server.create('log', { id: job.id });
+
+  visit('/travis-ci/travis-web/jobs/' + job.id);
+  waitForElement('#log > .log-line');
+
+  andThen(() => {
+    assert.equal(document.title, 'Job #1234.1 - travis-ci/travis-web - Travis CI');
+
+    assert.equal($('head link[rel=icon]').attr('href'), getFaviconUri('green'), 'expected the favicon data URI to match the one for passing');
+
+    assert.equal(jobPage.message, 'Pull Request #1 draft', 'displays message');
+    assert.equal(jobPage.badge, 'draft', 'displays badge');
+    assert.equal(jobPage.state, '#1234.1 passed', 'displays build number');
+
+    assert.equal(jobPage.createdBy.href, '/testuser');
+    assert.equal(jobPage.createdBy.text, 'Mr T');
+    assert.ok(jobPage.createdBy.avatarSrc.startsWith('/images/favicon-gray.png'));
+
+    assert.equal(jobPage.log, 'Hello log');
+    assert.notOk(jobPage.hasTruncatedLog);
+    assert.equal(jobPage.rawLogUrl, `${config.apiEndpoint}/v3/job/${job.id}/log.txt`);
+  });
+});
```

Comment:
I would suggest to just add an assertion for `draft` badge to existing test instead of creating a new one just for this small part - similar to the previous acceptance test in this PR. What do you think?

Reply:
@artursmirnov, I've added this one, because there is no test for pull request build, only for branch build